src/convex/ConvexAvatarUtils.sol

```diff
@@ -0,0 +1,43 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import {BaseAvatarUtils} from "../BaseAvatarUtils.sol";
+import {ConvexConstants} from "./ConvexConstants.sol";
+
+contract ConvexAvatarUtils is BaseAvatarUtils, ConvexConstants {
+    ////////////////////////////////////////////////////////////////////////////
+    // INTERNAL VIEW
+    ////////////////////////////////////////////////////////////////////////////
+    function getFxsAmountInFrax(uint256 _fxsAmount) internal view returns (uint256 usdcAmount_) {
+        uint256 fxsInUsd = fetchPriceFromClFeed(FXS_USD_FEED, CL_FEED_DAY_HEARTBEAT);
+        // Divisor is 10^8 and uint256 max ~ 10^77 so this shouldn't overflow for normal amounts
+        usdcAmount_ = (_fxsAmount * fxsInUsd) / FEED_DIVISOR_FXS_USD;
+    }
```

Comment:
Maybe add a note here that this is assuming frax:usd is 1:1

Reply:
comment added in the assumption for that method in 8f1285e8799cd5cb413b8cb597f1829926afab18